config/initializers/oembed_providers.rb

```diff
@@ -7,7 +7,8 @@
 purl_provider << 'https://purl.stanford.edu/*'
 purl_provider << 'http://searchworks.stanford.edu/*'
 
-purl_uat_provider = OEmbed::Provider.new('http://sul-purl-uat.stanford.edu/embed.{format}?&hide_title=true')
+purl_uat_provider = OEmbed::Provider.new('https://sul-purl-uat.stanford.edu/embed.{format}?&hide_title=true')
+purl_uat_provider << 'http://sul-purl-uat.stanford.edu/*'
```

Comment:
I don't think this addl. URL pattern is necessary since we don't pass `http` anywhere, but added here for consistency w/ the production URLs above.